src/app/ui/components/tooltip/basic-tooltip.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactNode } from 'react';
+
+import * as RadixTooltip from '@radix-ui/react-tooltip';
+
+import { Tooltip } from './tooltip';
+
+interface BasicTooltipProps {
+  children: ReactNode;
+  label?: string;
+  disabled?: boolean;
+  side?: RadixTooltip.TooltipContentProps['side'];
+}
+
```

Comment:
I missed this in the PR but am wondering we could instead name `BasicTooltip` as `Tooltip`? 

I like the wrapping idea to avoid repeated code but if `BasicTooltip` is the only one we use maybe it should be just called `Tooltip`